observability/micrometer/src/main/java/org/wildfly/extension/micrometer/MicrometerExtensionLogger.java

```diff
@@ -67,4 +67,8 @@ public interface MicrometerExtensionLogger extends BasicLogger {
     @LogMessage(level = WARN)
     @Message(id = 11, value = "Micrometer has been enabled, but no endpoint has been configured. A No-op metrics registry has been configured.")
     void noOpRegistryChosen();
+
+    @LogMessage(level = WARN)
+    @Message(id = 12, value = "Additional metrics systems discovered while configuring Micrometer: %s. Please see the administration guide for more information.")
```

Comment:
@jasondlee 

1) Where is this covered in the administration guide?
2) This approach is problematic for downstream, where the documentation is completely different.

Reply:
Hrm. It looks like the only mention is here: https://docs.wildfly.org/35/Admin_Guide.html#MicroProfile_Telemetry

Reply:
I have update the documentation to give more information about the log message.